packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -506,6 +506,30 @@ const Breakouts = WebexPlugin.extend({
 
     return breakout;
   },
+
+  /**
+   * assign participants to breakout session
+   * @param {Object} params
```

Comment:
params needs a better description. These doc strings should let the user know how to call the function